python/build.py

```diff
@@ -18,15 +18,17 @@
     pass
 packages = glob.glob(os.path.join(here, '../packages/*'))
 
-section_names = {'colormaps and styles': 'Colormaps and styles', 
-           'specialty plots': 'Specialty plots',
-           'gui applications': 'GUI applications', 
-           'miscellaneous': 'Miscellaneous', 
-           'backends': 'Rendering backends', 
+section_names = {'colormaps and styles': 'Colormaps and Styles', 
+           'plotting utilities': 'Plotting Utilities',
+           'plot types': 'Plot Types', 
+           'gui applications': 'GUI Applications', 
+           'backends': 'Rendering Backends', 
```

Comment:
I am pretty sure we usually use sentence case instead of title case?

Reply:
I thought sentence case read kinda weird, and these are titles?

Reply:
We should have a style for this. But I'll note that pandas and numpy both use sentence case below h1, and I think we should as well.

Reply:
fwiw I Prefer Sentence Case As Title Case Is Just Weird.